onset_maproom/maproom.py

```diff
@@ -102,33 +102,33 @@ def adm_borders(shapes):
     df["the_geom"] = df["the_geom"].apply(
         lambda x: x if isinstance(x, MultiPolygon) else MultiPolygon([x])
     )
+    index = df.index.tolist()
     shapes = df["the_geom"].apply(shapely.geometry.mapping)
+    for i in index: #this adds the district layer as a label in the dict
```

Comment:
Minor stylistic point: I'm pretty sure you could write this as
```
for i in df.index:
```
and eliminate line 105.